src/ExadelMentorship.WebApi/Middlewares/ExceptionHandlingMiddleware.cs

```diff
@@ -0,0 +1,31 @@
+﻿using ExadelMentorship.BusinessLogic.Exceptions;
+using System.Net;
+
+namespace ExadelMentorship.WebApi.Middlewares
+{
+    public class ExceptionHandlingMiddleware : IMiddleware
+    {
+        public async Task InvokeAsync(HttpContext context, RequestDelegate next)
+        {
+            try
+            {
+                await next(context);
+            }
+            catch (NotFoundException ex)
+            {
+                context.Response.StatusCode = (int)HttpStatusCode.NotFound;
```

Comment:
Use [`StatusCodes`](https://docs.microsoft.com/en-us/dotnet/api/microsoft.aspnetcore.http.statuscodes?view=aspnetcore-6.0) instead, so you don't have to cast to `int`.

Reply:
done